METRICS.md

```diff
@@ -84,6 +84,7 @@ the samples are reset.
 | Channel name              | Additional tags | Description
 | ------------------------- | --------------- | -----------
 | dispatch_aggregator       | aggregator_id   | Channel to dispatch metrics to a specific aggregator.
+| dispatch_receiver         | aggregator_id   | Channel to dispatch singular metrics to a given aggregator.
```

Comment:
I think I borked up the meaning of this channel, let me go fix it,

Reply:
I'd suggest changing the channel names for both, `dispatch_aggregator` is less accurate now because the term could cover either. `dispatch_aggregator_batch` and `dispatch_aggregator_map` maybe.

Reply:
That makes sense to me 👍 

Though, the usage of the two channels is still a mystery to me.

Reply:
There's two paths to the aggregators:
- UDP receiver -> []*Metric -> a bunch of PipelineHandler.DispatchMetrics -> Aggregator via dispatch_aggregator_batch
- HTTP receiver -> *MetricMap -> a bunch of PipelineHandler.DispatchMetricMap -> Aggregator via dispatch_aggregator_map

There's also the Consolidator who's job it is to receive []*Metric and *MetricMaps, bundle them all up in to a single *MetricMap, and then send them out somewhere.  Currently it's only used for HTTP forwarding, but it could easily be placed in front of the UDP receiver, which is part 1 of removing DispatchMetrics from the pipeline entirely.

Part 2 is dealing with the cloud provider, although it's basically the same thing, just drop a Consolidator in there.